SU-HSDO/mrc_blt
# Upgrade blt

# READY FOR REVIEW

# Summary
- Upgraded drush and blt to version 9.x

# Needed By (Date)
- End of sprint

# Urgency
- Medium

# Steps to Test

1. checkout this branch
2. composer update --prefer-source
3. ensure your blt commands still work
4. ensure drush still works as expected with version 9.

# Related
https://github.com/SU-HSDO/stanford_mrc/pull/101

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
